test/RibbonAutocallVault.ts

```diff
@@ -1999,27 +1936,27 @@ function behavesLikeRibbonOptionsVault(params: {
           returnAmt.sub(premiumAmount).toString()
         );
 
-        // user transfers the deposit amount and receives 3 coupons earned
+        // user transfers the deposit amount
         const earnedAmt = premiumAmount.div(numTotalObs).mul(3);
         assert.equal(
           userUSDCBalBefore.sub(userUSDCBalAfter).toString(),
-          depositAmount.sub(earnedAmt).toString()
+          depositAmount.toString()
         );
 
```

Comment:
can we check that price per share increases?

Reply:
there is a check on `accountVaultBalance` on these tests which is correctly increasing and uses the price per share in the calculation. seemed more complete given that it checks that both pps is increasing and that is increasing by the correct amount. In any case happy to add a more explicit check with just pps

PS: pps behavior by design may not be linearly up over the trade cycle - eg:

t0: vault is empty
t1: user deposits -> eg. pps = 100
t2: premium is paid mm to vault -> pps = 125
t3: call commitAndClose with 40% premium returning to MM -> pps = 115

so it does increase from t1->t3 (but if there are any returned coupons - this case 40% were returned - pps can possibly reduce from t2->t3)